datalad/core/distributed/tests/test_clone.py

```diff
@@ -262,14 +266,10 @@ def test_clone_isnot_recursive(src, path_nr, path_r):
 
 
 @slow  # 23.1478s
```

Comment:
what timing do you observe now? may be without testrepos it got speedy and does not need `@slow`?

Reply:
7.3s on a cold run of just this single test:

`python -m nose -s -v datalad.core.distributed.tests.test_clone:test_clone_into_dataset`

What is the desired threshold for slow?

Reply:
Removed

Reply:
```
def slow(f):
    """Mark test as a slow, although not necessarily integration or usecase test

    Rule of thumb cut-off to mark as slow is 10 sec
    """
    return attr('slow')(f)
```
I will push some quick tune up to CONTRIBUTING.md to reflect that

Reply:
pushed 193bd76b19beaf02fcdef984febdec58f1cbcb38 to maint